src/Chat/Maximized.js

```diff
@@ -172,7 +179,20 @@ const Maximized = ({
             isOwn={message.own}
             key={message.id}
           >
-            <Bubble isOwn={message.own}>
+            <Bubble
+              isOwn={message.own}
+              className={clsx({
```

Comment:
Eu to reparando que estamos usando o clsx e o classnames no projeto, melhor definirmos um

Reply:
Sim, mas acho que ai seria uma tarefa para fazermos depois: migrar para um unico.. E logico, definir qual é melhor (se tem melhor né).. Não precisamos entrar nesse mérito agora.. mas seria bom definir logo e migrar tudo e começar a usar um..